homeassistant/components/paperless_ngx/entity.py

```diff
@@ -0,0 +1,42 @@
+"""Paperless-ngx base entity."""
+
+from __future__ import annotations
+
+from homeassistant.components.sensor import EntityDescription
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import PaperlessConfigEntry, PaperlessCoordinator
+
+
+class PaperlessEntity(CoordinatorEntity[PaperlessCoordinator]):
+    """Defines a base Paperless-ngx entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        entry: PaperlessConfigEntry,
+        coordinator: PaperlessCoordinator,
+        description: EntityDescription,
+    ) -> None:
+        """Initialize the Paperless-ngx entity."""
+        CoordinatorEntity.__init__(
+            self,
+            coordinator=coordinator,
+        )
+
+        self.entry = entry
+        self.entity_description = description
+        self._attr_unique_id = f"{entry.entry_id}_{description.key}"
+
+        self._attr_device_info = DeviceInfo(
+            entry_type=DeviceEntryType.SERVICE,
+            identifiers={(DOMAIN, entry.entry_id)},
```

Comment:
```suggestion
        coordinator: PaperlessCoordinator,
        description: EntityDescription,
    ) -> None:
        """Initialize the Paperless-ngx entity."""
        CoordinatorEntity.__init__(
            self,
            coordinator=coordinator,
        )

        self.entity_description = description
        self._attr_unique_id = f"{self.coordinator.config_entry.entry_id}_{description.key}"

        self._attr_device_info = DeviceInfo(
            entry_type=DeviceEntryType.SERVICE,
            identifiers={(DOMAIN, self.coordinator.config_entry.entry_id)},
```
the entry is in the coordinator already anyway

Reply:
`coordinator` not `self.coordinator`